src/lib/components/workflow/workflows-summary-configurable-table.svelte

```diff
@@ -18,8 +21,8 @@
   let customizationDrawerOpen: boolean = false;
 
   $: ({ namespace } = $page.params);
-  $: columns = $workflowTableColumns?.[namespace] ?? [];
-  $: empty = workflows.length === 0 || columns.length === 0;
+  $: columns = $workflowTableColumns?.[namespace] ?? getDefaultColumns();
```

Comment:
What do you think about also checking that `JSON.parse(stringifiedOldColumns)` in `getDefaultColumns` has `length` and, if not, setting it to `DEFAULT_COLUMNS`? That way if a user currently has `workflow-table-columns` set to an empty an array (due to a previous edge case) they'll see default columns instead.